docs/Developers/Writing_Good_Commits.rst

```diff
@@ -96,10 +96,10 @@ Example:
 
 ::
 
-    When installing the .coafile to distutils.sysconfig.get_python_lib, we
-    ignore that this is not the installation directory in every case. Thus
-    it is easier, more reliable and platform independent to let distutils
-    install it by itself.
+    This entrypoint ensures that coala discovers
+    the bears correctly.
+    It helps not writing more functions inside
+    ``coalib`` for this.
```

Comment:
Also the lines should be 72 characters long for the commit description. These look shorter